services/github/github-total-star.tester.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Total Stars (User)')
+  .get('/hemantsonu20/total-stars.json')
+  .expectBadge({
+    label: 'Total Stars',
+    message: Joi.number().integer().positive(),
```

Comment:
Would be good to just use the `nonNegativeInteger` for consistency in these

Reply:
I used `isMetric` validator here, which makes more sense.